src/cell.cpp

```diff
@@ -706,31 +797,34 @@ Cell::to_hdf5(hid_t cell_group) const
 
 BoundingBox Cell::bounding_box_simple() const {
   BoundingBox bbox;
-  for (int32_t token : rpn_) {
-    //bbox &= model::surfaces[abs(token)-1].bounding_box(token > 0);
-    bbox &= model::surfaces[std::abs(token)-1].bounding_box(token > 0);
+  for (int32_t token : region_) {
+    bbox &= model::surfaces[abs(token) - 1].bounding_box(token > 0);
```

Comment:
What was the deal with `abs` vs `std::abs` here?

Reply:
Good question! It looks like at some point fairly early in the port I thought it was necessary to switch everything to `std::abs()` (see dcd603a). My best guess is that this was a workaround for a compiler bug. Still -- seems worthwhile to switch it back to std::abs for consistency perhaps? The change to `abs()` in this PR to this branch was not intentionally done, it was just picking up what was there from the cherry pick of the original PR's merge commit.